src/category_theory/Cat.lean

```diff
@@ -0,0 +1,41 @@
+/-
+Copyright (c) 2019 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+
+# Category of categories
+
+This file contains definition of category `Cat` of all categories.  In
+this category objects are categories and morphisms are functors
+between these categories.
+
+## Implementation notes
+
+Though `Cat` is not a concrete category, we use `bundled` to define
+its carrier type.
+-/
+import category_theory.concrete_category
+
+universes v u
+
+namespace category_theory
+
+/-- Category of categories. -/
+def Cat := bundled category.{v u}
+
+/-- Category structure on `Cat` -/
+instance Cat.category : category.{(max u v)+1 (max v (u+1))} Cat.{v u} :=
+{ hom := λ C D, @functor C.α C.str D.α D.str,
```

Comment:
Can you provide the instance so it is possible to just write `C.α ⥤ D.α` here?

Reply:
Done.